clouddriver-google/src/main/java/com/netflix/spinnaker/clouddriver/google/compute/InstanceTemplates.java

```diff
@@ -61,11 +61,13 @@ public GoogleComputeOperationRequest<Compute.InstanceTemplates.Insert> insert(
     return requestFactory.wrapOperationRequest(request, "insert");
   }
 
-  public PaginatedComputeRequest<Compute.InstanceTemplates.List, InstanceTemplate> list() {
+  public PaginatedComputeRequest<Compute.InstanceTemplates.List, InstanceTemplate> list(
+      String view) {
```

Comment:
Is this api used anywhere else?  I see the one spot below in the abstract caching agent that passes a fixed value of "FULL", but no place else this list call is updated?

Reply:
IF it's only ever called in that ONE operation - why not just add that as a fixed value here?

Reply:
In the clouddriver-google module there is only one call for this API. I wanted to leave it flexible but as it is only one call it's pointless. I will make the "FULL" view as the default one.